### PR TITLE
Finalize Request #8244

### PR DESCRIPTION
**Pull request by unl-creq**
Updating bulletin section.

### CODE REVIEW

**Comment by tlemburg** [comment]
Looks good

File: data/2015/majors/Natural Resource & Environmental Economics.xhtml [hunk 1]
```diff
@@ -19,34 +19,51 @@
...
 <p class="content-box-h-1">DESCRIPTION</p>
-<p class="faculty-list"><span class="faculty-list-bold">Coordinator: Karina Schoengold,</span> Department of Agricultural Economics, 307A Filley Hall</p>
-<p class="faculty-list"><span class="faculty-list-bold">Curriculum Committee:</span> Lynne</p>
+<p class="faculty-list-bold">Department Head: Van Tassell</p>
+<p class="faculty-list"><span class="faculty-list-bold">Professors:</span> Aiken, Azzam, Fulginiti, Giannakas, Hanson, Perrin, Peterson, Royer</p>
+<p class="faculty-list"><span class="faculty-list-bold">Associate Professors:</span> Parsons, Stockton, Yiannaka</p>
+<p class="faculty-list"><span class="faculty-list-bold">Assistant Professors:</span> Banerjee, Gustafson, Lubben, Mattos, Nogueira, Schoengold</p>
+<p class="faculty-list"><span class="faculty-list-bold">Instructors:</span> Barrett, Mattos, Peterson</p>
+<p class="faculty-list"><span class="faculty-list-bold">Coordinator: </span>Karina Schoengold, Department of Agricultural Economics, 307A Filley Hall</p>
+<p class="faculty-list"><span class="faculty-list-bold"></span> </p>
```
Comment (kabel) [inline data/2015/majors/Natural Resource & Environmental Economics.xhtml]: Remove this empty element.

File: data/2015/majors/Natural Resource & Environmental Economics.xhtml [hunk 2]
```diff
@@ -19,34 +19,51 @@
...
+<p class="requirement-sec-3">AGRO 131 &amp; AGRO 132 Plant Science &amp; Lab (HORT 131)</p>
+<p class="requirement-sec-3">BIOS 101 &amp; BIOS 101L General Biology &amp; Lab</p>
+<p class="requirement-sec-3">BIOS 109 General Botany</p>
+<p class="requirement-sec-3">BIOS 112 &amp; BIOS 112L Intro to Zoology &amp; Lab</p>
+<p class="requirement-sec-3">ENTO 115 Insect Biology (BIOS 115) &amp; ENTO 116 Insect Identification (BIOS 116)</p>
+<p class="requirement-sec-3">FDST 131 &amp; FDST 131L The Science of Food &amp; Lab (CHEM 131/NUTR 131)</p>
+<p class="requirement-sec-3">HORT 131 &amp; HORT 133 Plant Science &amp; Horticultural Plant Science Lab</p>
+<p class="requirement-sec-3">LIFE 120 &amp; LIFE 120L Fundamentals of Biology I &amp; Lab</p>
+<p class="requirement-sec-3">LIFE 121 &amp; LIFE 121L Fundamentals of Biology II &amp; Lab</p>
+<p><span class="requirement-bold"></span></p>
```
Comment (kabel) [inline data/2015/majors/Natural Resource & Environmental Economics.xhtml]: Remove this empty element.

File: data/2015/majors/Natural Resource & Environmental Economics.xhtml [hunk 3]
```diff
@@ -104,18 +121,18 @@
...
 <p class="requirement-sec-3">SOIL 153 Soil Resources (AGRO 153/HORT 153) (4 cr)</p>
 <p class="requirement-sec-3">WATS 281 Introduction to Water Science (GEOG 281/NRES 281) (3 cr)</p>
 <p class="requirement-sec-3">METR 100 Weather &amp; Climate (4 cr)</p>
 <p class="requirement-sec-3">NRES 208 Applied Climate Sciences (3 cr)</p>
 <p class="requirement-sec-1">Resource and Economics Requirements 31</p>
-<p class="requirement-sec-2">AECN 100 New Student Career Orientation 1 </p>
+<p class="requirement-sec-2">AECN 100 New Student Career Orientation 1</p>
 <p class="requirement-sec-2">AECN 265 Resource &amp; Environmental Economics I (NREE 265) 3</p>
 <p class="requirement-sec-2">AECN 357 Natural Resource &amp; Environmental Law (NREE 357) 3</p>
 <p class="requirement-sec-2">Capstone<span class="requirement-ital"> – </span>AECN 445 Agricultural &amp; Natural Resource Policy Analysis (NREE 445)<span class="requirement-ital"> (ACE 10) </span> 3</p>
```
Comment (kabel) [inline data/2015/majors/Natural Resource & Environmental Economics.xhtml]: Inappropriate use of ital span.

**Pull request merged**